libs/product/src/selectors/composite-product/composite-product.selectors.ts

```diff
@@ -206,9 +213,19 @@ function getMaxPricesIncludingOptionalItems(product: DaffCompositeProduct): Daff
 function getAppliedOptionsForConfiguration(product: DaffCompositeProduct, configuration: Dictionary<DaffCompositeConfigurationItem>): Dictionary<DaffCompositeProductItemOption> {
 	return (<DaffCompositeProduct>product).items.reduce((acc, item) => ({
 		...acc,
-		[item.id]: configuration[item.id] && configuration[item.id].value ? {
+		[item.id]: configuration[item.id] ? {
 			...item.options.find(option => option.id === configuration[item.id].value),
-			quantity: configuration[item.id].qty
+			quantity: (configuration[item.id].qty === null || configuration[item.id].qty === undefined) ? 1 : configuration[item.id].qty
```

Comment:
Can this be simplified to `configuration[item.id].qty || 1`?

Reply:
I wanted to allow for a zero quantity, so that wouldn't work